bin/convection

```diff
@@ -52,20 +55,24 @@ module Convection
     option :stacks, :type => :array, :desc => 'A ordered space separated list of stacks to diff'
     option :verbose, :type => :boolean, :aliases => '--v', :desc => 'Show stack progress'
     option :'very-verbose', :type => :boolean, :aliases => '--vv', :desc => 'Show unchanged stacks'
+    option :cloudfiles, :type => :array, :default => %w(Cloudfile)
+    option :delayed_output, :type => :boolean, :desc => 'Delay output until operation completion.', :default => true
     def diff(stack = nil)
-      @cloud.configure(File.absolute_path(options['cloudfile'], @cwd))
-      @cloud.diff(stack, stack_group: options[:stack_group], stacks: options[:stacks], &method(:emit_events))
+      task('diff', stack)
+      exit 1 if @errors
     end
 
-    desc 'print STACK', 'Print the rendered template for STACK'
-    def print(stack)
-      @cloud.configure(File.absolute_path(options['cloudfile'], @cwd))
+    desc 'print_template STACK', 'Print the rendered template for STACK'
+    option :cloudfile, :type => :string, :default => 'Cloudfile'
+    def print_template(stack)
```

Comment:
hehe, run into `print` overriding `Kernel#print`? 😜 I like `print-template` more anyways. Let's use hyphen over underscore in the documentation. Thor makes underscores interchangeable for dashes in commands as well as in flags. 👍